app/views/layouts/application.html.erb

```diff
@@ -19,6 +19,7 @@
         <ul>
           <li><a href="/shelters">Shelters</a></li>
           <li><a href="/pets">Pets</a></li>
+          <li>Favorites(<%= Pet.favorite_count %>)</li>
```

Comment:
Love that the logic for favorite count lives in the Pet model and not in the view!